src/main/java/com/mathworks/ci/MatlabBuild.java

```diff
@@ -33,14 +31,14 @@
     default ProcStarter getProcessToRunMatlabCommand(FilePath workspace,
             Launcher launcher, TaskListener listener, EnvVars envVars, String matlabCommand, String uniqueName)
             throws IOException, InterruptedException {
-        // Get node specific tmp directory to copy matlab runner script
-        String tmpDir = getNodeSpecificTmpFolderPath(workspace);
-        FilePath targetWorkspace = new FilePath(launcher.getChannel(), tmpDir);
+        // Get node specific temp .matlab directory to copy matlab runner script             
+        FilePath targetWorkspace = new FilePath(launcher.getChannel(),
+                workspace.getRemote() + "/" + MatlabBuilderConstants.TEMP_MATLAB_FOLDER_NAME);

```

Comment:
Does this file separator need to be platform specific?

Reply:
nope Filepath takes care of it if its just linux style.